generator/internal/mixin_utils.cc

```diff
@@ -49,10 +49,8 @@ std::unordered_map<std::string, std::string> const& GetMixinProtoPathMap() {
   return *kMixinProtoPathMap;
 }
 
-void ParseHttpRule(std::string const& selector,
-                   YAML::detail::iterator_value const& rule,
-                   google::api::HttpRule& http_rule) {
-  http_rule.set_selector(selector);
+google::api::HttpRule ParseHttpRule(YAML::detail::iterator_value const& rule) {
+  google::api::HttpRule http_rule;
```

Comment:
I think we dropped the `selector` field. we should set it in here if it is present.

Reply:
I tried to parse MethodDescriptor from proto, `method.options().GetExtension(google::api::http).selector()` is empty, so I think it is OK to leave `selector` empty here.

Reply:
I also think it is okay to omit the selector field. My instinct is that `ParseHttpRule` should parse the `HttpRule` completely, even if one of the fields is ignored. Up to you, though.

Reply:
Yeah, that is reasonable, I add the selector when it appears in the map of YAML.